web/src/pages/Settings/Config/Config.jsx

```diff
@@ -53,11 +53,17 @@ const LMSModeHandler = () => {
 
 const Config = ({ onClose }) => {
     const [file, setFile] = React.useState([]);
+    const [file_picked, setFilePicked] = React.useState(false);
```

Comment:
ngl, the mixing of camelCase and snake_case gives me the ick, but that's definitely just a nitpick.

Reply:
woops, just saw that, should be fixed now